src/style.css

```diff
@@ -0,0 +1,266 @@
+/*
+    #E7E3AF
+    #CCAB6F
+    #7D5535
+    #662F18
+    #761919
+  */
```

Comment:
- [OPTIONAL] Kindly remove commented section to have a cleaner file and follow best practices.

Reply:
@ZeenatLawal. Thanks for the suggestion.